Initializes the Community Ads RFP (#271)
readthedocs/ethicalads.io: content/posts/2022-community-ads-rfp.md

Title: Community Ads Program - Request for Proposals

> Date: May 13, 2022

Reviewer: ```suggestion
Date: May 16, 2022
```

description: Requests new proposals for Open Source projects to be featured across EthicalAds
tags: community, ads
authors: Ra Cohen
image: /images/posts/2022-community-ads-rfp.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@presetbase?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Presetbase Lightroom Presets</a> on <a href=" https://unsplash.com/photos/QN6NkYi3CKs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

Here at EthicalAds, we believe advertising can be used as a force for good, supporting open source and fostering community among devs across the world.

## Seeking proposals for our Community Ads Program

> As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives.

Reviewer: ```suggestion
As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives like open source conferences.
```

Do you run a conference or are starting a local chapter of a developer group that you'd like to advertise?
[Let us know](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp)!

## Previous Community Ads

We've been running free ads for community events for years now in a variety of languages.
Some examples of ads we've run before:

<div class="postimage text-center">
  <img src="{static}../images/pages/community-ads-pycon-example.png" alt="Want to give a talk at Pycon US 2022? Proposals open now until Dec 20th. Apply today!" style="max-width: 300px">
  <p>An ad raising awareness for the Call for Proposals for Pycon US 2022.</p>
</div>

<div class="postimage text-center">
  <img src="{static}../images/posts/2022-pycamp.png" alt="¡Programa tus ideas sin límites durante todo un fin de semana all-inclusive!" style="max-width: 300px">
  <p>An ad for a Spanish Python conference, PyCamp 2022.</p>
</div>

<div class="postimage text-center">
  <img src="{static}../images/posts/2019-PyBay.png" alt="Support your SF Bay Area PyCon. Join 750+ Pythonistas in San Francisco Aug 15-18" style="max-width: 300px">

>   <p>An ad for a San Fran Python conference from 2019.</p>

Reviewer: ```suggestion
  <p>An ad for a Bay Area Python conference from 2019.</p>
```

No self-respecting Californian calls it San Fran =)

</div>

As you can see, these are often ads for regional technical conferences but they don't have to be.
We'd love to run more ads supporting the developer community either by raising awareness for dev-focused charities or non-profit organizations.

## Know of a worthy cause?

> Spread the word by submitting a [community ad application](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp)today!

Reviewer: ```suggestion
Spread the word by submitting a [community ad application](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp) today!
```

Title: Community Ads Program - Request for Proposals
Date: May 10, 2022
description: Requests new proposals for Open Source projects to be featured across EthicalAds
tags: community, ads
authors: Ra Cohen
image: /images/posts/2022-community-ads-rfp.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@presetbase?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Presetbase Lightroom Presets</a> on <a href=" https://unsplash.com/photos/QN6NkYi3CKs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

> Here at EthicalAds, we believe advertising can be used as a force for good, supporting Open Source and fostering community among devs across the world.

Reviewer 2: We should decide is open source is cap'd, we don't do it consistently.

Reviewer: I do feel like this intro isn't exactly on-message. What exactly our message should be isn't precisely defined but generally:

* Community ads are for where paid ads don't make sense.
* Community ads MUST benefit the developer community.
* This means conferences and projects mostly.

## Seeking proposals for our Community Ads Program

As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives.
This is called our [Community Ads Program](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp).

> Historically, we've been proactive about reaching out to causes that we personally attend like PyCon, but we'd love to support other initiatives.

Reviewer: Actually PyCon mostly doesn't need advertising. Instead, we did regional Python conferences where the word is harder to get out.

Hence this blog post!

Do you have an open source project that is entirely free to use that you'd like to share with the world?

> Do you run a conference or are starting a local chapter of a developer group that you'd like to advertise?

Reviewer 2: These feel like they could be a bullet point, if we reworded them a bit.

> Let us know!

Reviewer 2: How? This should be a link probably..

> ## Previous Community Ads

Reviewer 2: These ads are different sizes & aspect ratios. I'd try and keep then consistent, or at least the horizontal ones the same size.

Reviewer: You can force them to be consistent by adding `style="max-width: 300px"` to the `img` tag.

<div class="postimage text-center">
  <img src="{static}../images/pages/community-ads-pycon-example.png" alt="Want to give a talk at Pycon US 2022? Proposals open now until Dec 20th. Apply today!">
  <p>An ad raising awareness for the Call for Proposals for Pycon US 2022.</p>
</div>

<div class="postimage text-center">
  <img src="{static}../images/posts/2022-pycamp.png" alt="¡Programa tus ideas sin límites durante todo un fin de semana all-inclusive!">
  <p>An ad for a Spanish Python conference, PyCamp 2022.</p>
</div>

<div class="postimage text-center">
  <img src="{static}../images/posts/2019-PyBay.png" alt="Support your SF Bay Area PyCon. Join 750+ Pythonistas in San Francisco Aug 15-18">
  <p>An ad for a San Fran Python conference from 2019.</p>
</div>

As you can see, these are often ads for regional technical conferences but they don't have to be!
We'd love to run more ads supporting the developer community either by raising awareness for dev-focused charities or non-profit organizations.
Just reach out!

## Know of a worthy cause?

Do these examples remind you of your own cause? Spread the word by submitting a community ad application today!

> * [Launch the application!](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp)

Reviewer 2: This is an odd CTA, and a single bullet point feels odd. There's a better way to do this.

Author: reworked into the copy

Thanks again for being along with us on this journey to build an ethical ad network.

> Please [email us](mailto:ads@ethicalads.io) if you have any ideas or feedback on our product or roadmap,

Reviewer 2: Roadmap doesn't make sense here. Seems like this sign off was copied from the newsletter?
